setup.py

```diff
@@ -16,7 +16,7 @@ def main():
         all_deps += extras[group_name]
     extras['all'] = all_deps
 
-    base_libs = ['neptune-client>=0.9.8']  # TODO: update requirements
+    base_libs = ['neptune-client>=0.9.8', 'fastai']
```

Comment:
Probably some future version of `neptune-client` will be required.
The one which will be deployed, with updated `neptune.new.integrations` module.

Let's call it 0.9.17 and keep an eye on `neptune-client` releases.